config-keyvault/pom.xml

```diff
@@ -7,15 +7,22 @@
     <parent>
         <groupId>com.azure.microprofile</groupId>
         <artifactId>azure-microprofile-parent</artifactId>
-        <version>1.0.0-beta.1</version>
+        <version>1.0.0-beta.2</version>
         <relativePath>../parent/pom.xml</relativePath>
     </parent>
     
     <groupId>com.azure.microprofile</groupId>
     <artifactId>azure-microprofile-config-keyvault</artifactId>
-    <version>1.0.0-beta.1</version>
+    <version>1.0.0-beta.2</version>
+    <packaging>jar</packaging>
+
     <name>Azure extension for MicroProfile :: Config :: Key Vault</name>
     <description>Azure extension for MicroProfile Key Vault Custom ConfigSource.</description>
+    <url>https://docs.microsoft.com/java/azure/microprofile</url>
+
```

Comment:
I recall that Reza is not so much a fan of that page. Let us use https://aka.ms/java/ee instead.

Reply:
Good catch. Updated to https://aka.ms/java/ee in commit 3699dc6.